actions/check-product-changes/index.js

```diff
@@ -30,7 +32,11 @@ async function main(params) {
     // this might not be updated and action execution could be permanently skipped
     // a ttl == function timeout is a mitigation for this risk
     await stateMgr.put('running', 'true', { ttl: 3600 });
-    return await poll(params, { stateLib: stateMgr, filesLib });
+    const activationResult = await poll(params, { stateLib: stateMgr, filesLib });
+    // send activation result to observability endpoint
+    await observabilityClient.sendActivationLog(activationResult);
```

Comment:
can't we run the observability client as individual, scheduled action that collects the activation logs and sends them to the observability worker? kind of a sidecar?